lib/galaxy/webapps/galaxy/api/folder_contents.py

```diff
@@ -232,7 +239,19 @@ def _load_folder_contents(self, trans, folder, include_deleted):
                 #         subfolder.api_type = 'folder'
                 #         content_items.append( subfolder )
 
-        for dataset in folder.datasets:
+        datasets_size = int(len(folder.datasets))
```

Comment:
len() should already return you an int()

Reply:
thanks, I will check